build.sbt

```diff
@@ -237,18 +195,14 @@ lazy val `haskell-integration-tests` = project
   .in(file("modules/haskell-integration-tests"))
   .settings(noPublishSettings)
   .settings(haskellIntegrationTestSettings)
-  .dependsOn(server, `client-netty`, channel, fs2)
+  .dependsOn(server, `client-netty`, `rpc-service`, fs2)
```

Comment:
Same here, `rpc-service` is a transitive dependency of the other dependencies

Reply:
👍  https://github.com/higherkindness/mu-scala/pull/900/commits/585bbc2172172de2b4ea77dc14580fbc4b8e2d4d